bin/nic.pl

```diff
@@ -186,6 +186,12 @@
 		}
 	}
 	untie(@lines);
+} else {
+	# New project, so create .gitignore
+	my $gitignore = "$dirname/.gitignore";
+	open my $fileHandle, ">>", $gitignore or exitWithError("Failed to create $gitignore.");
+	print $fileHandle ".theos/\npackages/\n";
```

Comment:
Do we want to ignore `.DS_Store` as well?

Reply:
Yeah, probably a good idea to.